src/main/resources/static/api.yaml

```diff
@@ -88,4 +141,88 @@ components:
       properties:
         hipId:
           type: string
-          description: Identifier of the health information provider where patient record needs to be looked up.
\ No newline at end of file
+          description: Identifier of the health information provider where patient record needs to be looked up.
+
+    ConsentRequestsRepresentation:
+      type: object
+      properties:
+        limit:
+          type: integer
+        size:
+          type: integer
+        offset:
+          type: integer
+        requests:
+          type: array
+          maxItems: 100
+          items:
+            $ref: '#/components/schemas/ConsentRequestRepresentation'
+
+    ConsentRequestRepresentation:
+      type: object
+      properties:
+        purpose:
+          type: object
+          properties:
+            description:
+              type: string
+        hitypes:
+          type: array
+          items:
+            $ref: '#/components/schemas/HITypeRepresentation'
+        requester:
```

Comment:
should we rename requester as "practitioner"? https://www.hl7.org/fhir/practitioner.html
Also the ID should be "Identifier" object, and "Identifier" object should have the actual value, type of identifier and additionally/optionally system (which is an URL). Also, for practitioner/request, should we allow for multiple identifiers? that way, a request can have information of possibly referring to multiple systems (NHRR Id, MCI/NCI/DCI Id, other affiliation id).